### ISSUE TITLE
[v2] no rules on HTTPS listener

### ISSUE THREAD

**Issue by somdoron** [kubernetes-sigs/aws-alb-ingress-controller#1485]
The ingress is configured to listen on both HTTP and HTTPS, checking on the AWS console the rules for the HTTP listener are populated correctly but HTTPS rules are empty. 
When I remove HTTP from `listen-ports` everything works correctly.
 
The dex ingress is currently the only one in the group.

spec:
```yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: dex
  namespace: dex  
  annotations:
    alb.ingress.kubernetes.io/group.name: internal
    kubernetes.io/ingress.class: alb
    alb.ingress.kubernetes.io/listen-ports: '[{"HTTP": 80}, {"HTTPS": 443}]' #'[{"HTTPS": 443}]' 
    alb.ingress.kubernetes.io/scheme: internet-facing
    alb.ingress.kubernetes.io/target-type: ip
    alb.ingress.kubernetes.io/actions.ssl-redirect: '{"Type": "redirect", "RedirectConfig": { "Protocol": "HTTPS", "Port": "443", "StatusCode": "HTTP_301"}}'    
    alb.ingress.kubernetes.io/healthcheck-path: /healthz
  labels:
    app: dex
spec:
  rules:    
    - host: dex.example.com
      http:
        paths:
          - path: /*
            backend:
              serviceName: ssl-redirect
              servicePort: use-annotation          
          - path: /*
            backend:
              serviceName: dex
              servicePort: 80
```

**Comment by M00nF1sh** [comment]
this will be fixed by https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/1484

**Comment by M00nF1sh** [comment]
closing this since it's addressed by [v2.0.0-rc3 release](https://github.com/kubernetes-sigs/aws-alb-ingress-controller/releases/tag/v2.0.0-rc3) :D.
feel free to reopen if further issue exists.
